Base/usr/share/man/man8/WebServer.md

```diff
@@ -1,6 +1,6 @@
 ## Name
 
-WebServer
+WebServer - Serenity web server
```

Comment:
...eh.
doesn't sound that useful or good, it's literally the program name

Reply:
> doesn't sound that useful or good, it's literally the program name
All the man pages are like that. Did it for consistency.